packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.Props.ts

```diff
@@ -54,9 +54,11 @@ export interface IComboBoxProps extends ISelectableDroppableTextProps<IComboBox>
 
   /**
    * Whether the ComboBox auto completes. As the user is inputing text, it will be suggested potential matches from the list of options. If
-   * the combo box is expanded, this will also scroll to the suggested option, and give it a selected style. Defaults to false.
+   * the combo box is expanded, this will also scroll to the suggested option, and give it a selected style.
+   *
+   * @default "on"

```

Comment:
NIT: change this to single quotes instead of double quotes

Reply:
👍  Would you like me to also change 
- https://github.com/IanVS/office-ui-fabric-react/blob/e151da5520b367108ec4bd30c50d79b493e98c74/packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.Props.ts#L66, 
- https://github.com/IanVS/office-ui-fabric-react/blob/e151da5520b367108ec4bd30c50d79b493e98c74/packages/office-ui-fabric-react/src/components/Panel/Panel.Props.ts#L56, 
- and https://github.com/IanVS/office-ui-fabric-react/blob/e151da5520b367108ec4bd30c50d79b493e98c74/packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.Props.ts#L19 

so that it's consistently single quotes?

Reply:
I can also do that in a separate PR, because it seems there is also some inconsistency between `@default`, `@defaultvalue`, and `@defaultValue` (the last of which actually causes `Value` to be displayed on the website).  If you let me know which is preferred (`@default` or `@defaultvalue`), I'd be happy to open another PR.

Reply:
`@default` is preferred and I would say don't muddy up this change with changing other double quote usages. At most you could open a bug around making the code consistent